data/apply-planet_disputed_area_features_name_suppression.sql

```diff
@@ -0,0 +1,84 @@
+        (st_transform(st_geomfromtext('POLYGON ((-171.33 -11.30,-171.33 -10.80,-170.81 -10.80,-170.81 -11.30,-171.33 -11.30))', 4326), 3857)),
+        (st_transform(st_geomfromtext('POLYGON ((110.58 7.31,110.58 10.23,116.2 10.23,116.2 7.31,110.58 7.31))', 4326), 3857)),
+        (st_transform(st_geomfromtext('POLYGON ((113.20 9.39,113.20 11.95,116.28 11.95,116.28 9.39,113.20 9.39))', 4326), 3857)),
+        (st_transform(st_geomfromtext('POLYGON ((117.4 14.8,117.4 15.49,118.11 15.49,118.11 14.8,117.4 14.8))', 4326), 3857)),
+        (st_transform(st_geomfromtext('POLYGON ((-88.37 16.05,-88.37 16.22,-88.20 16.22,-88.20 16.05,-88.37 16.05))', 4326), 3857)),
+        (st_transform(st_geomfromtext('POLYGON ((123.3 25.66,123.3 25.99,123.81 25.99,123.81 25.66,123.3 25.66))', 4326), 3857)),
+        (st_transform(st_geomfromtext('POLYGON ((110.81 15.35,110.81 17.41,113.28 17.41,113.28 15.35,110.81 15.35))', 4326), 3857)),
+        (st_transform(st_geomfromtext('POLYGON ((-75.262 18.16,-75.262 18.64,-74.7 18.64,-74.7 18.16,-75.262 18.16))', 4326), 3857)),
+        (st_transform(st_geomfromtext('POLYGON ((9.37 0.80,9.37 0.81,9.38 0.81,9.38 0.80,9.37 0.80))', 4326), 3857)),
+        (st_transform(st_geomfromtext('POLYGON ((171.08 -22.78,171.08 -22.05,172.36 -22.05,172.36 -22.78,171.08 -22.78))', 4326), 3857)),
+        (st_transform(st_geomfromtext('POLYGON ((42.43 -17.31,42.43 -16.79,43.03 -16.79,43.03 -17.31,42.43 -17.31))', 4326), 3857)),
+        (st_transform(st_geomfromtext('POLYGON ((-66.50 80.81,-66.50 80.8,-66.39 80.8,-66.39 80.81,-66.50 80.81))', 4326), 3857)),
+        (st_transform(st_geomfromtext('POLYGON ((149.31 45.25, 148.51 46.28, 145.16 43.85, 146.08 43.21, 149.31 45.25))', 4326), 3857))
+   ) as b (geom)),
+
+ suppress_point as
+     (select *
+      from planet_osm_point l,
+           bbox
+      where st_within(l.way, bbox.geom)),
+
+ update_point as
+     (update planet_osm_point p
+         set tags = delete(p.tags, (select array_agg(key) from (select distinct skeys(tags) as key from suppress_point) a where key like '%name%'))
+         from suppress_point s
+         where s.osm_id = p.osm_id),
+
+ suppress_line as
+     (select *
+      from planet_osm_line l,
+           bbox
+      where st_within(l.way, bbox.geom)),
+
+ update_line as
+     (update planet_osm_line p
+         set tags = delete(p.tags, (select array_agg(key) from (select distinct skeys(tags) as key from suppress_line) a where key like '%name%'))
+         from suppress_line s
+         where s.osm_id = p.osm_id),
+
+ suppress_polygon as
+     (select *
+      from planet_osm_polygon l,
+           bbox
+      where st_within(l.way, bbox.geom)),
+
+ update_polygon as
+     (update planet_osm_polygon p
+         set tags = delete(p.tags, (select array_agg(key) from (select distinct skeys(tags) as key from suppress_polygon) a where key like '%name%'))
+         from suppress_polygon s
+         where s.osm_id = p.osm_id),
+
+ suppress_roads as
```

Comment:
Do we dip into the roads table? Is this one necessary?

Reply:
I wasn't sure if we use it, I just ran it on all tables for completeness. I can remove it, if its unneeded.

Reply:
Please remove this (as it would otherwise add processing time needlessly).

I don't see us using the roads table, in places like https://github.com/tilezen/vector-datasource/blob/master/data/apply-schema-update.sql or elsewhere.

We _do_ dip into the rel (relations) table for road shield information (but not for name values). No action needed there.

Reply:
Sounds good. I removed that bit